### ISSUE TITLE
CI build `sudo npm install -g markdown-link-check` started failing with `Cannot find module 'cheerio'`

### ISSUE THREAD

**Issue by mieubrisse** [tcort/markdown-link-check#203]
Hey folks, thanks so much for markdown-link-check! We've been happily using it in our CI to verify our docs links for about a year, but the build this morning started failing with:

```
Error: Cannot find module 'cheerio'
Require stack:
- /usr/local/lib/node_modules/markdown-link-check/node_modules/html-link-extractor/index.js
- /usr/local/lib/node_modules/markdown-link-check/node_modules/markdown-link-extractor/index.js
- /usr/local/lib/node_modules/markdown-link-check/index.js
- /usr/local/lib/node_modules/markdown-link-check/markdown-link-check
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:15)
    at Function.Module._load (node:internal/modules/cjs/loader:763:27)
    at Module.require (node:internal/modules/cjs/loader:991:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/markdown-link-check/node_modules/html-link-extractor/index.js:3:17)
    at Module._compile (node:internal/modules/cjs/loader:1102:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1131:10)
    at Module.load (node:internal/modules/cjs/loader:967:32)
    at Function.Module._load (node:internal/modules/cjs/loader:807:14)
    at Module.require (node:internal/modules/cjs/loader:991:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/local/lib/node_modules/markdown-link-check/node_modules/html-link-extractor/index.js',
    '/usr/local/lib/node_modules/markdown-link-check/node_modules/markdown-link-extractor/index.js',
    '/usr/local/lib/node_modules/markdown-link-check/index.js',
    '/usr/local/lib/node_modules/markdown-link-check/markdown-link-check'
  ]
}
```

It looks like there was a new version of markdown-link-check released 3 hours ago; did that change something?

**Comment by palootcenas-outreach** [comment]
Seems like the issue is in [html-link-extractor](https://www.npmjs.com/package/html-link-extractor) that doesn't state `cheerio` in it's dependencies, but uses it. Also it's GitHub page was canceled.

**Comment by tcort** [comment]
Should be fixed in v3.10.2
